src/Cmf/MainBundle/Resources/data/posts/2015-04-01.yml

```diff
@@ -0,0 +1,62 @@
+name: "cmf-hackday-may-2015"
+parent: "/news"
+title: "CMF Hackday 1st/2nd May 2015"
+label: false
+format: markdown
+template: CmfMainBundle:Cms:news_detail.html.twig
+publish_start_date: "2015-04-01"
+body: |
+  We are happy to announce a joint Symfony CMF / Sulu.io hackday in Dornbirn on the 1st and 2nd of May. Join us
+  for a hacking, discussion and workshops (if there is interest).
+
+  If you would like to come, please register [here](<http register here dot com>)
+
+  ### Schedule
+  
+  #### Friday 1st May
+
+  09:00 - Breakfast
+  09:30 - <workshop> / <talk> / <discussion> / <hacking>
+  12:00 - Lunch
+  12:30 - <workshop> / <talk> / <discussion> / <hacking>
+  18:00 - Mountain dinner at the [Karren](http://www.karren.at/?id=1800)
+
+  #### Saturday 2nd May
+
+  09:00 - Breakfast
+  09:30 - <workshop> / <talk> / <discussion> / <hacking>
+  12:00 - Lunch
+  12:30 - <workshop> / <talk> / <discussion> / <hacking>
+
```

Comment:
i think that list sounds good. i would say like this:

* we have discussion panels according to interest, with topics like cmf v2, jackalope v2 and sulu + cmf. have a free text field where people can write about their interest. or maybe ask people to send to the mailinglist if they have discussion topics, so we can prepare a bit.
* if there are at least 3 people interested, we offer a free tutorial on sulu / on the cmf

then i can do the cmf tutorial with newbies, if we manage to attract any. or you can do a sulu tutorial if there are people interested in that.